src/ucs/datastruct/list.h

```diff
@@ -114,13 +114,30 @@ static inline int ucs_list_is_empty(ucs_list_link_t *head)
     return head->next == head;
 }
 
+/**
+ * @return Whether the list contains only elem.
+ */
+static inline int
+ucs_list_is_first(ucs_list_link_t *head, ucs_list_link_t *elem)
```

Comment:
minor: const for both params here and below

Reply:
it would make it inconsistent with other list functions, so IMO better not add it